scripts/publish-docs.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+#
+# Triggers mapzen docs build to publish to https://mapzen.com/documentation/.
+
+if [[ ${PERFORM_RELEASE} ]]
```

Comment:
Haha it does not yet! :) I will be adding a script to perform the release using Circle's API similar to [this one we use for Eraser Map](https://github.com/mapzen/eraser-map/blob/master/scripts/perform-release.sh) as part of https://github.com/mapzen/android/issues/279.